ppr-ui/src/views/AuthStub.vue

```diff
@@ -8,23 +8,23 @@
       div
         v-btn(class="form-primary-btn", @click="letUserIn", color="primary") Let me in!
 </template>
-<script>
-    import AuthHelper from '@/utils/auth-helper'
 
-    export default {
-        components: {},
+<script lang="ts">
+  import {createComponent} from "@vue/composition-api";
+  import {Data} from "@vue/composition-api/dist/ts-api/component";
+  import AuthHelper from '@/utils/auth-helper'
 
-        data() {
-            return {}
-        },
+  export default createComponent({
+    setup(): Data {
+      function letUserIn(): void {
+        AuthHelper.authFake()
```

Comment:
Question for the team, do we prefer this syntax or arrow functions for inline functions. My personal preference is arrow functions, so it could be refactored to:

`const letUserIn = () => AuthHelper.authFake()`

I'm open to discussion though.  This PR has a bit of a mix, so it would be good to settle on an approach.